packages/application-components/src/components/modal-pages/utils/modal-page-top-bar.tsx

```diff
@@ -77,14 +76,9 @@ const ModalPageTopBar = (props: Props) => {
         display: flex;
         align-items: center;
         justify-content: space-between;
-        padding: ${appKitDesignTokens.paddingForModalTopBar};
-        background-color: ${props.color === 'neutral'
-          ? appKitDesignTokens.backgroundColorForPageHeader
-          : uiKitDesignTokens.colorSurface};
-        border-bottom: 1px solid
-          ${props.color === 'neutral'
-            ? uiKitDesignTokens.colorSurface
-            : appKitDesignTokens.borderColorForModalTopBarWhenSurface};
+        padding: ${uiKitDesignTokens.spacing40} ${uiKitDesignTokens.spacing55} 0;
+        background-color: ${uiKitDesignTokens.colorSurface};
```

Comment:
One question here. Do we no longer differentiate styling between 'neutral' and 'surface' for this component?

Reply:
Thanks for noticing.

We don't need to differentiate anymore since the token we're removing (`borderColorForModalTopBarWhenSurface`) had the `colorSurface` value.